src/nt_users_tool/excel_processing.py

```diff
@@ -0,0 +1,82 @@
+from openpyxl import Workbook, load_workbook
+from datetime import date
+
+from constants import *
+
+
+def create_results_workbook(excel_desired_path: str, sheets_names_list: list):
+    """Creates the output excel file according to arguments
+
+    Args:
+        excel_desired_path (str): Path where the output file should be located
+        sheets_names_list (list): Names of the sheets inside the file
+    """
+    workbook = Workbook()
+    workbook.remove(workbook.active)
```

Comment:
You might not want to do that! In case the output file is the same as the input file, and you just want to add /overwrite the ouptut worksheets

Reply:
Okay. This was done in case the output file was not the same as the input.